bin/node-template/node/src/command.rs

```diff
@@ -184,11 +189,32 @@ pub fn run() -> sc_cli::Result<()> {
 				let task_manager =
 					sc_service::TaskManager::new(config.tokio_handle.clone(), registry)
 						.map_err(|e| sc_cli::Error::Service(sc_service::Error::Prometheus(e)))?;
+
+				let info_provider = |_, maybe_prev_info: Option<(InherentData, Digest)>| async {
+					const BLOCKTIME_MILLIS: u64 = 2 * 3_000;
```

Comment:
I vaguely think this should be `Timestamp::MinimumPeriod`.

Reply:
yes, it should be, but I don't have access to pallet constants here, do I? if not, I can add an explaining comment here